src/main/resources/static/codegen/built-cas2-api-spec.yml

```diff
@@ -2268,11 +2268,35 @@ components:
         description:
           type: string
           example: 'More information about the application has been requested from the POM (Prison Offender Manager).'
+        statusDetails:
```

Comment:
Just above here in the OpenAPI spec should we include the created_at timestamp of the Cas2ApplicationStatus so we can give that to the frontend to present?

Reply:
that would be good, but it would also mean adding to this big PR 🙈 should be done as part of this ticket? https://dsdmoj.atlassian.net/jira/software/c/projects/CAS2/boards/1347?selectedIssue=CAS2-213

Reply:
oh actually, this model doesn't have a `created_at` because it's the seeded data, it's the `Cas2StatusUpdate` which has the timestamp, and that is already being returned and rendered on the timeline as it stands